RELEASE_NOTES.md

```diff
@@ -0,0 +1,80 @@
+* Kubernetes: Disable kubernetes's pod eviction [#2124](https://github.com/Microsoft/pai/pull/2124)
+* Pylon: WebHDFS library compatibility [#2134](https://github.com/Microsoft/pai/pull/2134)
+* Hadoop: Extend nm expiry time to 60 mins [#2142](https://github.com/Microsoft/pai/pull/2142)
+* Alart Manager: Make it more clear in service not up [#2105](https://github.com/Microsoft/pai/pull/2105)
+* Grafana: Use yarn's metrics in cluster view [#2148](https://github.com/Microsoft/pai/pull/2148)
+* Web Portal: Allow jsonc in job submission [#2084](https://github.com/Microsoft/pai/pull/2084)
+
+### Command ###
+
+* Build: Add error message when image build failed [#2133](https://github.com/Microsoft/pai/pull/2133)
+* Deploy: Only restart docker deamon, if the configuration is updated. [#2138](https://github.com/Microsoft/pai/pull/2138)
+
+### Documentation ###
+
+* Update document about docker data root's configuration [#2052](https://github.com/Microsoft/pai/pull/2052)
+* Add issue to readme [#2044](https://github.com/Microsoft/pai/pull/2044)
+* Note tell user take care dev-box-version for doc [#2087](https://github.com/Microsoft/pai/pull/2087)
+* HDFS data migration doc [#2096](https://github.com/Microsoft/pai/pull/2096)
+
+### Examples ###
+
+* Add /usr/local/cuda/extras/CUPTI/lib64 to LD_LIBRARY_PATH [#2043](https://github.com/Microsoft/pai/pull/2043)
+* Add an exmaple of horovod with rdma & intel mpi [#2112](https://github.com/Microsoft/pai/pull/2112)
+
+## Bug Fixes ##
+
+* Issue [#2099](https://github.com/Microsoft/pai/pull/2099) is fixed by
+  * Launcher: Revise the definition of Framework running state [#2135](https://github.com/Microsoft/pai/pull/2135)
+  * REST server: Classify two states to WAITING [#2154](https://github.com/Microsoft/pai/pull/2154)
+
+## Upgrading from Earlier Release ##
+
+* Download the code package of release v0.10.0 from [release page](https://github.com/Microsoft/pai/releases),
+  or you can clone the code by running:
+
+  ```bash
+  git clone --branch v0.10.0 git@github.com:Microsoft/pai.git
+  ```
+
+* Prepare your cluster configuration by instructions in [OpenPAI Configuration](./docs/pai-management/doc/how-to-write-pai-configuration.md).
+  In the *service-configuration.yaml* file, configure the docker info as following:
+
+  ```yaml
+  docker-namespace: openpai
+  docker-registry-domain: docker.io
+  docker-tag: v0.10.0
+  ```
+
+* In the code source directory, upgrade by following steps:
+
+  ```bash
+  # Stop the services
+  python paictl.py service stop -p cluster_configuration_file_path
+  # After the services are stopped, stop the kubernetes cluster
+  python paictl.py cluster k8s-clean -p cluster_configuration_file_path
+  # Reboot the kubernetes cluster
+  python paictl.py cluster k8s-bootup -p cluster_configuration_file_path
+  # Push cluster configuration file to kubernetes
+  python paictl config push -p cluster_configuration_file_path
```

Comment:
We will need to upgrade the config and push it before run `service stop` and `cluster k8s-clean`.
I will come out with a more consistent upgrade process soon.

Reply:
I've found out a upgrade command list that may works. I did not run the commands one by one because some commands seems useless were also run when I try to upgrade.

```
  # Docker exec into 0.9 dev-box

$ vim /cluster-confuiguration/services-configuration.yaml
  # change docker-tag

$ git checkout pai-0.9.y

$ python paictl.py service stop

$ python paictl.py k8s-clean -p /cluster-configuration

$ git checkout pai-0.10.y

$ python deployment/configMigration.py /cluster-configuration /new-cluster-configuration
  # copied from ./deployment/tools/ to aroung the python import bug

  # Adjust the configurations
$ mkdir /cluster-configuration/old
$ mv /cluster-configuration/*.yaml /cluster-configuration/old
$ mv /new-cluster-configuration/*.yaml /cluster-configuration/

$ python paictl.py check -p /cluster-configuration

$ python paictl.py k8s-bootup -p /cluster-configuration

$ python paictl.py config push -p /cluster-configuration

$ python paictl.py service start
```